operator/v1/types_console.go

```diff
@@ -45,13 +45,15 @@ const (
 	// Branding for OpenShift
 	BrandOpenShift Brand = "openshift"
 	// Branding for The Origin Community Distribution of Kubernetes
-	BrandOKD       Brand = "okd"
+	BrandOKD Brand = "okd"
 	// Branding for OpenShift Online
-	BrandOnline    Brand = "online"
+	BrandOnline Brand = "online"
 	// Branding for OpenShift Container Platform
-	BrandOCP       Brand = "ocp"
+	BrandOCP Brand = "ocp"
 	// Branding for OpenShift Dedicated
 	BrandDedicated Brand = "dedicated"
+	// Branding for OpenShift Azure
```

Comment:
Nit:  product name is `Azure Red Hat OpenShift`

Reply:
updated.